eZ/Publish/Core/FieldType/RichText/RichTextStorage/Gateway/DoctrineStorage.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\FieldType\RichText\RichTextStorage\Gateway;
+
+use eZ\Publish\Core\Persistence\Doctrine\DoctrineConnection;
+use eZ\Publish\Core\FieldType\RichText\RichTextStorage\Gateway;
+use RuntimeException;
+
+class DoctrineStorage extends Gateway
+{
+    /**
+     * @var \eZ\Publish\Core\Persistence\Doctrine\DoctrineConnection
```

Comment:
Can we type hint against the DBAL class?
Afaik our own here is implementation detail, as it looks like we can optionally remove it once we are back to standard pgsql sequences right?